src/ucp/core/ucp_worker.inl

```diff
@@ -122,6 +122,16 @@ ucp_worker_extract_request_by_id(ucp_worker_h worker, ucs_ptr_map_key_t id)
     return request;
 }
 
+static UCS_F_ALWAYS_INLINE int
+ucp_worker_keepalive_is_enabled(ucp_worker_h worker)
+{
+    ucs_assert((worker->context->config.keepalive_interval == 0) ||
+               ((worker->context->config.keepalive_interval != 0) &&
+                (worker->context->config.ext.keepalive_num_eps != 0)));
```

Comment:
better just assert worker->context->config.ext.keepalive_num_eps in the keepalive worker function

Reply:
done